### ISSUE TITLE
Service (factory) naming inconsistencies

### ISSUE THREAD

**Issue by ciwchris** [johnpapa/angular-styleguide#471]
I would expect a consistent naming usage in the examples instead of a mix, ex: 'test', 'testservice', 'testService'. Are such inconsistencies acceptable, on even intended?

- logger: [y010](https://github.com/johnpapa/angular-styleguide#style-y010)
- dataservice: [y033](https://github.com/johnpapa/angular-styleguide#style-y033)
- dataService: [y052](https://github.com/johnpapa/angular-styleguide#style-y052)
- creditService: [y035](https://github.com/johnpapa/angular-styleguide#style-y035)
- showServiceError: [y035](https://github.com/johnpapa/angular-styleguide#style-y035)
- exception: [y111](https://github.com/johnpapa/angular-styleguide#style-y111)
- movieService: [y081](https://github.com/johnpapa/angular-styleguide#style-y081)
- moviesPrepService:  [y081](https://github.com/johnpapa/angular-styleguide#style-y081)
- storage: [y010](https://github.com/johnpapa/angular-styleguide#style-y010)
- storageService: [y100](https://github.com/johnpapa/angular-styleguide#style-y100)
- localstorage: [y152](https://github.com/johnpapa/angular-styleguide#style-y152)
- avengerService: [y100](https://github.com/johnpapa/angular-styleguide#style-y100)

**Comment by johnpapa** [comment]
I hate camelCase in JavaScript. but it exists.

Feel free to PR the casing if you like

**Comment by robrothedev** [comment]
I think in the end it's up to the developer to determine a consistent naming convention from the suggestions provided in the style guide.  Many online examples I've seen seem to lack consistency.

I've adopted these conventions:

```html
// modules
videoApp

// controllers
VideoListCtrl as video_list_ctrl
VideoCtrl
EditVideoCtrl

// services
Videos

// directives
videoList

// constants - uppercase from my PHP background
VIDEO_API_URL
```

**Comment by ciwchris** [comment]
Yes, in the end developer consistency is what's important. I think it would be ideal to be a good example by being consistent in the guide as well. I'll clean the usage up and submit a PR at which time more discussion can take place if needed.

**Comment by johnpapa** [comment]
naming conventions are in the style guide already. Feel free to use those and clean up any inconsistencies you see.

**Comment by johnpapa** [comment]
addressed with #476 to fix the oddities.

services that are clear should omit `service` as a suffix. Often these are verbs. Services that are nouns and could be confused with models (i.e. customers vs customersService) should have a suffix.

exception and showCreditError and some others were inline functions, not anything angular related.
